src/main/webapp/user-private-comments.jsp

```diff
@@ -56,12 +56,9 @@
             <form action="/user-profile" method="POST">
               <input type="hidden" name="taskID" value="<%=task.getTaskID()%>"> 
               <br>
-              <%
-                String ID = (String) "box-" + task.getTaskID();
-              %>
-              <div id="box-<%=task.getTaskID()%>" onload="init('<%=ID%>')" type="text"></div>
-              <input type="hidden" name="message-<%=task.getTaskID()%>" value="document.getElementById('<%=ID%>').value(outputData);"</input>
-              <button onclick="save()" class="deep-button submit-button" type="submit" class="deep-button">Update Comment</button>
+              <div id="box-<%=task.getTaskID()%>" onload="init(<%=task.getTaskID()%>)" type="text">Here</div>
+              <button onclick="save(init(<%=task.getTaskID()%>))" class="deep-button submit-button" type="submit" class="deep-button">Update Comment</button>
+              <input type="hidden" name="message-<%=task.getTaskID()%>" value="save()"</input>
```

Comment:
After save is called onclick of the button above, populate the input value with the value of save (which I think should be outputData?

Reply:
yep.  line [37 specifically](https://github.com/googleinterns/step171-2020/pull/94/files/badc7c173a80c0589f4702beac544d07cd686fd4#diff-243abfa42a44d81d97be6c5af0c05900R37), instead of just console.log there, that would be the correct place to do `document.getElementById('message' + pcID).value(outputData)`